src/app/education-page/education-page.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-education-page',
+  templateUrl: './education-page.component.html',
+  styleUrls: ['./education-page.component.css']
+})
+export class EducationPageComponent {
+
+}
```

Comment:
Ten pusty komponent raczej niepotrzebny w tym pull requeście.

Reply:
Jeśli ten pull request jest work in progress i jeszcze będziesz tu coś dodawała, to przed nazwą pull requesta daj WIP:
np.
WIP: Feature/create projects and skills components